src/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -205,6 +249,49 @@ public static void TestECDsaPublicKey_ValidatesSignature()
             }
         }
 
+        [Theory, MemberData(nameof(BrainpoolCurves))]
+        public static void TestECDsaPublicKey_BrainpoolP160r1_ValidatesSignature(byte[] curveData, byte[] existingSignature)
+        {
+            // This signature was produced as the output of ECDsaCng.SignData.
+            //
+            // Note that since ECDSA signatures contain randomness as an input, this value is unlikely
+            // to be reproduced by another equivalent program.
```

Comment:
I think this comment was borrowed from a static input test and doesn't apply? After all, there's no signature here.